shopno/src/styles/style.scss

```
@@ -0,0 +1,594 @@
+    color: #505050;
+    text-shadow: 9px 9px 10px #0000003d;
+    &__row {
+        text-align: center;
+    }
+    &__accent {
+        background-color: #75f3e9;
+    }
+}
+
+.button {
+    padding: 13px 55px;
+    font-size: 1.8rem;
+    font-family: "Montserrat";
+    color: #000000b3;
+    text-transform: uppercase;
+    border: 2px solid #00000080;
+    border-radius: 2px;
+    background-color: rgba(113, 245, 235, 0.5);
+    transition: background-color 0.8s 0s;
+    &:hover {
+        cursor: pointer;
+        background-color: rgba(32, 219, 110, 0.918);
+    }
+}
+
+.about {
+    padding: 0 10px;
+    background: #f8f9fa;
+    &__container {
+        padding: 121px 0 165px 0;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        @media (max-width: 1024px) {
+            padding: 10vw 15px 10vw 15px;
+        }
+    }
+    &__block {
+        margin-top: 145px;
+        display: flex;
+        justify-content: space-between;
+        flex-wrap: wrap;
+        @media (max-width: 860px) {
+            margin-top: 40px;
+            justify-content: center;
+            
+        }
+    }
+    &__image-block {
+        flex-basis: 48.5%;
+        text-align: center;
+        @media (max-width: 860px) {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            flex-basis: 40%;
+        }
+        @media (max-width: 640px) {
+            flex-basis: 90%;
+        }
+    }
+    &__image {
+        height: auto;
+        @media (max-width: 860px) {
+            vertical-align: middle;
+            width: 95%;
+        }
+        @media (max-width: 640px) {
+            flex-basis: 90%;
+            max-width: 400px;
+        }
+    }
+    &__content-block {
+        flex-basis: 45%;
+        @media (max-width: 860px) {
+            margin-top: 4vw;
+            flex-basis: 90%;
+        }
+    }
+    &__content-block-title {
+        font-size: 2rem;
+        letter-spacing: 1.4px;
+        font-family: "Merriweather";
+        font-weight: bold;
+        text-transform: uppercase;
+        &:not(:first-child) {
+            margin-top: 34px;
+            font-size: 1.7rem;
+            letter-spacing: 1px;
+        }
+    }
+    &__content-block-text {
+        margin-top: 20px;
+        font-size: 1.6rem;
+        line-height: 175%;
+        letter-spacing: 0.7px;
+        font-family: "Raleway";
+        color:#000000bd
+    }
+}
+
+.works {
+    padding: 0 10px;
+    background: #ffffff;
+    &__container {
+        padding: 118px 0 128px 0;
+        text-align: center;
+        @media (max-width: 1024px) {
+            padding: 10vw 0 10vw 0;
+        }
+    }
+    &__examples {
+        margin-top: 118px;
+        @media (max-width: 1024px) {
+            margin-top: 4vw;
+        }
+    }
+}
+
+.examples {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    gap: 10px 15px;
+    &__item {
+        width: 32.456%;
```

`32.456%`???

---

This was  for pixel perfect.
```suggestion
        width: 32.5%;
```